### ISSUE TITLE
rseqc junction_saturation plot has known/novel curves swapped

### ISSUE THREAD

**Issue by matrs** [ewels/MultiQC#970]
**Description of bug:**
[multiqc_rseqc_saturation.tar.gz](https://github.com/ewels/MultiQC/files/3167835/multiqc_rseqc_saturation.tar.gz)

Individual plots from `RSeQC`'s `junction_saturation.py` (data displayed side by side, as the original in RSeQC plot), have the curve "All junctions" swapped with the "known junctions". Here 
 I attach the multiqc report/data and the original pdf and R script from  `RSeQC` for comparison.

**MultiQC run details (please complete the following):**
 - Command used to run MultiQC: `multiqc SRR5494629_rseqc-sat/  -o SRR5494629_multiqc`
 - MultiQC Version: version 1.7
 - Operating System: linux
 - Python Version: Python 2.7.15 and 3.6.3
 - Method of MultiQC installation: conda

 **Additional context**
I ran `junction_saturation.py` in multiple input files. I tried two different environments, one with python 2.7.15 and other with python 3.6.3 trying `RSeQC 2.6.4` and `3.0.0`, both with multiqc 1.7 and the results were the same.

I checked the source of `multiqc`  and with my very limited knowledge of it, I think these lines are correct when I compare them to the R script (the same z, y, w are used in the R script):

```python
self.junction_saturation_all[f['s_name']][v] = parsed['z'][k]
self.junction_saturation_known[f['s_name']][v] = parsed['y'][k]
self.junction_saturation_novel[f['s_name']][v] = parsed['w'][k]
```
so I suppose there is a problem in the `javascript` part of the code. 

Cheers

**Comment by ewels** [comment]
Hi @matrs,

Thanks for reporting this - I have a vague sense of deja-vu with the problem, which probably means that I half-fixed it in the past or something (I can't find anything in the code commits or issues though..). Anyway, I'll take a look when I get a chance 👍 

Phil

**Comment by ewels** [comment]
This issue was fixed in https://github.com/ewels/MultiQC/pull/903 and will be released in v1.8
